static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -148,6 +151,7 @@ export const replayEventMap: Record<ReplayEventKey, string | null> = {
   'replay.list-sorted': 'Sorted Replay List',
   'replay.list-time-spent': 'Time Spent Viewing Replay List',
   'replay.list-view-setup-sidebar': 'Views Set Up Replays Sidebar',
+  'replay.number_of_timeline_gaps': 'Number of Gaps in Replay Timeline',
```

Comment:
```suggestion
  'replay.gaps-detected': 'One or more Gaps in Replay Timeline',
```

Reply:
We also track if there are 0 gaps, but I do like gaps detected a lot more, will change that!

Reply:
Maybe we could track the max gap size too?

Reply:
that's a great idea!

Reply:
Or even avg, min too? Not sure what aggregate fxs amplitude offers

Reply:
If we're tracking 0 gaps too, I'm not sure my suggested names fit anymore

Reply:
```suggestion
  'replay.gaps-detected': 'Number of Gaps in Replay Timeline',
```